README.md

```diff
@@ -1 +1,150 @@
-# Hello-Microverse
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+Testing my skills on Linters, Gitflow and Pull Request
+
+**[Hello Microverse]** is a simply web page.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://stylelint.io/">Linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[It has a homepage]**
+- **[Header tag has color blue]**
+- **[SEO Friendly]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.   
+
+### Prerequisites
+
+In order to run this project you need:
+```
+1. Git installed on your machine.
+2. Any code editor [vscode preferred].
+2. Any browser [chrome preferred].
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone git@github.com:CaptainBawa/Hello-Microverse.git
+```
+
+### Install
+
+Install this project with:
+
+```nmp install/npm i
+```
```

Comment:
[OPTIONAL]  kindly consider adding some space  after opening  back-ticks and before the closing back-ticks  and this will make your readme more readable 
eg: 
``` 
npm install/npm i
```